### PR TITLE
Target Map Documentation

### PR DESCRIPTION
**Pull request by achimweigel**
**What this PR does / why we need it**:

Target Map Documentation.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc user
Target Map Documentation
```

### CODE REVIEW

File: docs/guided-tour/target-maps/03-multiple-subinst/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Now you can see successful installations on your Landscaper resource cluster:
+
+```
+kubectl get di -n cu-example                                                  
+NAME                                    TYPE                                            PHASE       EXPORTREF   AGE
+multiple-subinst                   Succeeded                                      17s
+multiple-subinst-sub-blue-hhdmz    Succeeded   multiple-subinst-sub-blue-hhdmz    16s
+multiple-subinst-sub-green-55nw2   Succeeded   multiple-subinst-sub-green-55nw2   16s
+multiple-subinst-sub-red-srm8g     Succeeded   multiple-subinst-sub-red-srm8g     16s
+```
```
Comment (robertgraeff) [inline docs/guided-tour/target-maps/03-multiple-subinst/README.md:100]: There is something wrong here. The text says "installations", the command says "di", and the result table is the same as above.

File: docs/guided-tour/target-maps/05-other-examples/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Other Target Map Examples
+
+## Target Map on Subinstallation Level
+
+The example defined [here](../04-forward-map) deploys the same config maps on the target cluster as the others. The
+difference is that it imports 3 targets as standard targets and in this 
+[Subinstallation](../04-forward-map/component/blueprint/sub/subinst.yaml) it converts these into a target map.
```
Comment (robertgraeff) [inline docs/guided-tour/target-maps/05-other-examples/README.md:7]: Just a proposal: should we move this readme into the 04 example? I was a bit confused why 04 had no readme.

**Review by robertgraeff — CHANGES_REQUESTED** [review]

File: docs/guided-tour/target-maps/03-multiple-subinst/README.md (hunk 1 above)
Comment (achimweigel) [reply docs/guided-tour/target-maps/03-multiple-subinst/README.md]: I have fixed this such that now the DIs are presented here.

File: docs/guided-tour/target-maps/05-other-examples/README.md (hunk 2 above)
Comment (achimweigel) [reply docs/guided-tour/target-maps/05-other-examples/README.md]: I have mode the part of the readme.

**Review by robertgraeff — APPROVED** [review]

**Pull request merged**